### PR TITLE
rosyama.ru

### PR DESCRIPTION
**Pull request by galeksandrp**
Issue author: @galeksandrp

### CODE REVIEW

File: src/chrome/content/rules/rosyama.ru.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+² refused
+⁴ self signed
+-->
+<ruleset name="rosyama.ru">
+	<target host="rosyama.ru" />
+
+	<target host="www.rosyama.ru" />
+
+	<rule from="^http://www\.rosyama\.ru/"
+		to="https://rosyama.ru/" />
```
Comment (J0WI) [inline src/chrome/content/rules/rosyama.ru.xml]: They serve different content

Comment (galeksandrp) [inline src/chrome/content/rules/rosyama.ru.xml]: Mismatch -> self signed -> empty response.

Comment (J0WI) [inline src/chrome/content/rules/rosyama.ru.xml]: On http://www.rosyama.ru/ I get a 404, on http://rosyama.ru/ I get redirected to https://rosyama.ru/

Comment (galeksandrp) [inline src/chrome/content/rules/rosyama.ru.xml]: Well, removed.

**Pull request merged**